shared/searchv3/results-list/index.desktop.js

```diff
@@ -23,6 +23,7 @@ class SearchResultsList extends Component<void, Props, void> {
         key={id}
         onClick={() => onClick(id)}
         onShowTracker={onShowTracker ? () => onShowTracker(id) : undefined}
+        selectedId={this.props.selectedId}
```

Comment:
I think you should do the `id === this.props.selectedId` here and pass in the selected state. This way all rows do not have their props change when the `selectedId` changes, and the rows are not concerned with what the selected id of the collection is.